src/api/Controllers/ExportController.cs

```diff
@@ -184,6 +191,63 @@ public async Task<IActionResult> ExportCsvAsync([FromQuery][MaxLength(MaxPageSiz
         return File(Encoding.UTF8.GetBytes(stringWriter.ToString()), "text/csv", "boreholes_export.csv");
     }
 
+    /// <summary>
+    /// Asynchronously gets all <see cref="Borehole"/> with attachments filtered by ids.
+    /// </summary>
+    /// <param name="ids">The required list of borehole ids to filter by.</param>
+    /// <returns>A ZIP-file including the borehole data as .json and all corresponding attachments.</returns>
+    [HttpGet("zip")]
+    [Authorize(Policy = PolicyNames.Viewer)]
+    public async Task<ActionResult> ExportJsonWithAttachmentsAsync([FromQuery][MaxLength(MaxPageSize)] IEnumerable<int> ids)
+    {
+        var idCount = ids.Count();
+        if (ids == null || idCount < 1) return BadRequest("The list of IDs must not be empty.");
+
+        try
+        {
+            var boreholes = await context.Boreholes.GetAllWithIncludes().AsNoTracking().Where(borehole => ids.Contains(borehole.Id)).ToListAsync().ConfigureAwait(false);
+            var files = await context.BoreholeFiles.Include(f => f.File).Where(f => ids.Contains(f.BoreholeId)).ToListAsync().ConfigureAwait(false);
+            var fileName = $"boreholes_with_attachments_{DateTime.UtcNow:yyyyMMddHHmmss}";
```

Comment:
PP: Wir sollten einheitliche Namen für die Exporte verwenden. Beim CSV-Export heisst das File z.B. `boreholes_export.csv`. Ich finde diese Variante mit dem Datumsstempel besser. Vorschlag:  Format des Filenames in allen Exporten wiederverwenden.